### PR TITLE
emba data

### PR DESCRIPTION
**Pull request by cwhittemore1**
🥇

### CODE REVIEW

File: app/pods/auth/login/template.hbs [hunk 1]
```diff
@@ -29,10 +29,14 @@
 
   {{#link-to 'auth.join'}}
     {{#form.footer}}
-      <div class="visible content">New to us?</div>
-      <div class="hidden content">
-        Join today!
-      </div>
+      {{#unless (media 'isMobile')}}
```
Comment (CygnusRoboticus) [inline app/pods/auth/login/template.hbs]: Please make the `/auth/join/template.hbs` file match this pattern.

File: app/pods/components/hero-message/template.hbs [hunk 2]
```diff
@@ -1,7 +1,4 @@
-<div class="ui message text container">
-  <div class="header">
-    Welcome to WebWars!
-  </div>
-  <p>We just updated our privacy policy here to better service our customers. We recommend reviewing the changes.</p>
+<div class="header">
```
Comment (CygnusRoboticus) [inline app/pods/components/hero-message/template.hbs]: Separate Task: This should be loaded from the backend for system messages.

File: app/pods/media/service.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import Service from '@ember/service';
+import MediaService from 'ember-responsive/services/media';
 import { computed } from '@ember/object';
 
-export default Service.extend({
+export default MediaService.extend({
```
Comment (CygnusRoboticus) [inline app/pods/media/service.js]: :+1:

File: package.json [hunk 4]
```diff
@@ -42,6 +42,7 @@
     "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^5.0.1",
+    "ember-responsive": "^3.0.0-beta.3",
```
Comment (CygnusRoboticus) [inline package.json]: Lock down these version numbers. `^3.0.0-beta.3` => `3.0.0-beta.3`.

File: app/pods/auth/login/template.hbs (hunk 1 above)
Comment (cwhittemore1) [reply app/pods/auth/login/template.hbs]: 💯

File: app/pods/auth/join/template.hbs [hunk 5]
```diff
@@ -38,10 +38,14 @@
...
-      <div class="hidden content">
-        Login now!
-      </div>
+      {{#unless (media 'isMobile')}}
+        <div class="visible content">Been here before?</div>
+        <div class="hidden content">
+          Login now!
+        </div>
+      {{else}}
+        <div class='content'>New to us?</div>
```
Comment (CygnusRoboticus) [inline app/pods/auth/join/template.hbs]: Typo

**Pull request merged**